src/Data/Weapons/Sword/SkywardBlade/index.tsx

```diff
@@ -39,7 +39,7 @@ const sheet: IWeaponSheet = {
     conditional: {
       value: condPassive,
       path: condPassivePath,
-      header: conditionalHeader(tr, icon, iconAwaken, trm("might")),
```

Comment:
why are we changing the conditional name to just "Conditional"?

Reply:
All the weapons have it as "Conditional" at the moment since I wasn't too sure on what we're going to name them so I just named it "Conditional" and conform all the weapons to it. Kinda felt like it looked more explanatory when naming them as "Conditional" rather than naming them under their ability/passive name. But I guess it's entirely up to you on how we're moving forward with this.
![image](https://user-images.githubusercontent.com/17585549/159210749-14adcd61-b151-4434-a859-3e37b6eb5cd5.png)
![image](https://user-images.githubusercontent.com/17585549/159210755-b1eeca8d-a4a5-44ea-8ac6-b5fe0ababf92.png)

Reply:
I rather not use the word "Conditional" since its a technical word used in the system. I think its fine to use the passive name. I don't see this change as an improvement.

Reply:
How about weapons that don't particularly have an ability name ie. Engulfing Lightning?
![image](https://user-images.githubusercontent.com/17585549/159211632-0e5c74aa-03fc-4ed5-b558-bfc789642f6d.png)

Should the conditional name just be left empty?

Reply:
That could just be "Timeless Dream: Eternal Stove" or empty

Reply:
Okay I'll make changes on all the weapon and make a new PR